docs/leaderboards.md

```diff
@@ -44,23 +44,23 @@ For details on the FM-backbones and instructions to replicate the results, check
-| **Number of steps**            | 12,500                    | 12,500 (2)                |
-| **Base batch size**            | 4,096 (1)                 | 32                        |
-| **Base learning rate**         | 0.01 (1)                  | 0.001                     |
-| **Early stopping**             | 5% * [Max epochs]         | 10% * [Max epochs] (3)    |
-| **Optimizer**                  | SGD                       | AdamW                     |
-| **Momentum**                   | 0.9                       | n/a                       |
-| **Weight Decay**               | 0.0                       | n/a                       |
-| **betas**                      | n/a                       | [0.9, 0.999]              |
-| **LR Schedule**                | Cosine without warmup     | Cosine without warmup     |
-| **number of patches per slide**| 1                         | dataset specific (4)      |
+|                                | WSI patch-level classification tasks | WSI slide-level classification tasks | WSI patch-level segmentation tasks |
+|--------------------------------|---------------------------|---------------------------|---------------------------|
+| **Backbone**                   | frozen                    | frozen                    | frozen                    |
+| **Head**                       | single layer MLP          | ABMIL                     | Mult-stage convolutional  |
+| **Dropout**                    | 0.0                       | 0.0                       | 0.0                       |
+| **Hidden activation function** | n/a                       | ReLU                      | n/a                       |
+| **Output activation function** | none                      | none                      | none                      |
+| **Number of steps**            | 12,500                    | 12,500 (2)                | 2,000                     |
+| **Base batch size**            | 4,096 (1)                 | 32                        | 64                        |
+| **Base learning rate**         | 0.01 (1)                  | 0.001                     | 0.0001                    |
+| **Early stopping**             | 5% * [Max epochs]         | 10% * [Max epochs] (3)    | 10% * [Max epochs] (3)    |
+| **Optimizer**                  | SGD                       | AdamW                     | AdamW                     |
+| **Momentum**                   | 0.9                       | n/a                       | n/a                       |
+| **Weight Decay**               | 0.0                       | n/a                       | n/a                       |
+| **betas**                      | n/a                       | [0.9, 0.999]              | [0.9, 0.999]              |
+| **LR Schedule**                | Cosine without warmup     | Cosine without warmup     | PolynomialLR              |
+| **number of patches per slide**| 1                         | dataset specific (4)      | dataset specific (4)      |
```

Comment:
Should we list the loss function here as well?

Reply:
yes, added